code/modules/mob/living/damage_procs.dm

```diff
@@ -73,7 +73,6 @@
 		if(JITTER)
 			if(status_flags & CANSTUN)
 				jitteriness = max(jitteriness,(effect * blocked))
-	updatehealth()

```

Comment:
not needed here, the update is called by the relevant effect procs when necessary.